rs/cross-chain/proposal-cli/src/canister/mod.rs

```diff
@@ -77,6 +79,9 @@ impl TargetCanister {
             TargetCanister::ExchangeRateCanister => {
                 "https://github.com/dfinity/exchange-rate-canister.git"
             }
+            TargetCanister::SolRpc => {
+                "https://github.com/internet-computer-protocol/sol-rpc-canister.git"
```

Comment:
Should be under the Dfinity Github org (Would you mind also changing the `EvmRpc` one to `https://github.com/dfinity/evm-rpc-canister.git`? Initially the EvmRpc repo was under the `internet-computer-protocol` Github organization but we moved it to Dfinity)
```suggestion
                "https://github.com/dfinity/sol-rpc-canister.git"
```

Reply:
Good catch! Done.